core/src/main/java/org/bitcoinj/core/listeners/PeerConnectionEventListener.java

```diff
@@ -21,21 +21,24 @@
 import java.util.Set;
```

Comment:
This file seems to be unchanged. If I'm right, can you remove the JavaDoc reformatting from this PR?